src/functional.d

```diff
@@ -151,13 +152,13 @@ The syntax to convert a tuple back to list is a bit hard to figure out.
 @name("Filter by index") @safe unittest{
```

Comment:
This segment does not show up on the page because it does not have a comment. I wanted to add one but I could not figure out what it was doing. :)